README.rst

```diff
@@ -87,17 +87,21 @@ Building the Driver on Linux and macOS
 Building the Driver on Windows
 ------------------------------
 
-.. |win-vs-version| replace:: VS16 8.0.16 
+.. |win-vs-version| replace:: VS16 
+.. |win-php-version| replace:: 8.1.18 
+
 
 **Note**: Snowflake supports only thread-safe versions of PHP.
 
+You must install Microsoft Visual Studio 2019 (VS16) or earlier with the C++ development installer option.
+
 To build the PHP driver for Windows:
 
 #. Download and install the PHP SDK:
```

Comment:
This step is download and install PHP, not PHP SDK. Remove the option of build PHP from source code and ask customer to always install PHP from official binary would be much easier for us as we don't have to teach customer how to build and install PHP from source code, which kind of out of our scope.

Reply:
Done